modular_skyrat/modules/blueshield/code/special.dm

```diff
@@ -1,24 +1,26 @@
 /obj/item/gun/energy/e_gun/revolver //The virgin gun.
-	name = "energy revolver"
-	desc = "An advanced energy revolver with the capacity to shoot both electrodes and lasers."
-	force = 7
-	ammo_type = list(/obj/item/ammo_casing/energy/electrode, /obj/item/ammo_casing/energy/laser/revolver)
+	name = "X-R12 energy revolver"
+	desc = "An experimental model advanced energy weapon with the capacity to shoot both electrodes and lasers, used by many private defense contractors from Romulus to New Moscow, its also rather heavy and would certainly hurt to get pistol whipped by."
+	force = 22
```

Comment:
This seems very, *very* powerful. Are you sure?

Reply:
I think the revolver should be comparable to the HoS' laser.

Reply:
It is comparable and the revolver isn't more powerful